_data/events.yml

```diff
@@ -2,8 +2,8 @@
   date: 2022-8-24
   link: https://digital.gov/event/2022/08/24/2022-federal-plain-language-summit/
   description: >-
-   The Summit brings together federal employees to highlight plain language successes in many federal agencies at all stages of the communications journey. Speakers address debunking myths, meeting varying audience’s needs, managing the review process and stakeholders, and supporting plain language editors.  
+   On August 24, 2022, our summit brought together federal employees to highlight plain language successes in many federal agencies at all stages of the communications journey. Experience the slides and some recordings from some of our speakers. Speakers addressed debunking myths, meeting varying audience\’s needs, managing the review process and stakeholders, and supporting plain language editors.  
```

Comment:
Removed the `\` in `aduience's`. It doesn't need to be escaped

**Before**
![image](https://user-images.githubusercontent.com/3385219/206757099-41e6d07d-4810-4d82-b185-0f3cc9de1015.png)

**After**
![image](https://user-images.githubusercontent.com/3385219/206757202-8612e63d-afba-4b8e-86ee-c0b65d3459c6.png)